2024/planning-process.md

```diff
@@ -0,0 +1,134 @@
+
+For example, if there are 20 remaining proposals, each organization will choose a number 1, number 2, down to number 20. Let’s say a proposal for flying cars gets the following ranks: 1, 5, 7, 10, 14, 20. That will give it the following points: 20, 15, 13, 10, 6, 1 = 65 points total.
+
+(If there appear to be quite a few proposals remaining in this round, the group can decide to only ask for each organization to submit their Top `X` number, while rating the rest zero. For example, if 50 proposals remain at this point, it likely makes sense to only submit a Top 25, and to not worry about ranking 26-50 in a specific order.)
+
+After Round 3, every remaining proposal can be listed in order, with the group’s highest priority at the top, and the lowest at the bottom.
+
+NOTE: in the past, “Exclude” was a way to express many different kinds of sentiments — including: does not meet the minimum qualifications, or this is low priority to us. We expect “veto” to be used much less than “exclude”, because there are other ways to express those sentiments. Vetoing is specifically for objecting to a particular proposal being included *at all,* no matter how highly ranked other organizations might find it. It’s for expressing objection to a particular web technology, perhaps to something an organization has already expressed an objection to in the web standards process or otherwise has an objection to implementing.
+
+## Interop 2023 Evaluation
+
+During the same period organizations are making Round 3 decisions, the organizing committee should evaluate the status of Interop 2023, and decide which Focus Areas will be retired, and which will be carried over.
+
+\[XXX]
+
+* * *
+
+# Decision point — what to do about Interop 2023?
+
+## There seem to be 3 options:
+
+### 1. Assume carryover.  Retire some.  Add how many ever that leaves space for.
+
+1. Start with the assumption that all Interop 2023 projects will be carried over — except for the ones that are “complete”
+    1. Figure out a process for determining what should be retired.
+        1. Perhaps by voting for each proposal one-by-one, to keep it or not (that’s what we did last year).
+        2. Perhaps by score — everything above 95% Interop gets retired, or something similar.
+    2. Decide how many total Focus Areas we want for Interop 2024. Perhaps 20? 15?
+    3. That determines how many “open slots” we have.
+        1. 26 - the number retired = the number carried over.
+        2. If we are aiming for 20 areas in 2024, and 18 are carried over (retiring 6)... then we have space to add 2 new areas.
+
+### 2. Automatically ‘reapply’ all Focus Areas
+
+1. Carryover nothing.
+2. Take all 26 Focus Areas from Interop 2023, and put them into the selection process.
+
+### 3. Make every Focus Area ‘reapply’
```

Comment:
I prefer this option.

I agree that previous focus areas should automatically be given weight by orgs when considering priorities. But I also think there should be a real assessment of the value of completing existing focus areas vs adding new ones. In some cases this will be obvious (e.g. if a feature is still missing from some implementations), and in some cases it won't be (e.g. if there's a few percent left, and work is needed to understand if the remaining work carries its weight vs other options). In some cases there might be a third way e.g. grouping the remaining work from several previous focus areas into a new focus area with just the remaining failures (e.g. one can imagine flexbox + grid + subgrid failures becoming a single "Layout modes" focus area).

Re-proposing is more work, but I think that work has enough value to make it worthwhile.